core/src/main/java/de/jplag/options/SimilarityMetric.java

```diff
@@ -1,14 +1,27 @@
 package de.jplag.options;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.function.ToDoubleFunction;
 
 import de.jplag.JPlagComparison;
+import de.jplag.Match;
 
 public enum SimilarityMetric implements ToDoubleFunction<JPlagComparison> {
     AVG("average similarity", JPlagComparison::similarity),
     MIN("minimum similarity", JPlagComparison::minimalSimilarity),
     MAX("maximal similarity", JPlagComparison::maximalSimilarity),
-    INTERSECTION("matched tokens", it -> (double) it.getNumberOfMatchedTokens());
+    INTERSECTION("matched tokens", it -> (double) it.getNumberOfMatchedTokens()),
+    SYMMETRIC("symmetric similarity", it -> {
```

Comment:
Isn't this equivalent to the implementation of the "average similarity", only ignoring base code?